server/index.js

```diff
@@ -1,15 +1,17 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const db = require('../database/index.js');
+const cors = require('cors');
 
 const app = express();
 const port = 3002;
 
-
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(cors({
+  origin: 'http//localhost:3002',
+}))
 app.use(express.static('public'));
```

Comment:
Do you not need express to parse JSON ?

Reply:
Express OR bodyParser was my understanding.

Reply:
Sorry if I led you astray. They do different things. It's all express. The bodyparser middleware helps parse JSON and urlencoded data. The cors middleware sets headers for cross-origin communications.

Reply:
yeah I don't think I need either in this case